specification/vmware/resource-manager/Microsoft.AVS/preview/2019-08-09-preview/vmware.json

```diff
@@ -0,0 +1,1277 @@
+    "Endpoints": {
+      "description": "Endpoint addresses",
+      "properties": {
+        "nsxtManager": {
+          "description": "Endpoint for the NSX-T Data Center manager",
+          "type": "string",
+          "readOnly": true
+        },
+        "vcsa": {
+          "description": "Endpoint for Virtual Center Server Appliance",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "IdentitySource": {
+      "description": "vCenter Single Sign On Identity Source",
+      "properties": {
+        "name": {
+          "description": "The name of the identity source",
+          "type": "string"
+        },
+        "alias": {
+          "description": "The domain's NetBIOS name",
+          "type": "string"
+        },
+        "domain": {
+          "description": "The domain's dns name",
+          "type": "string"
+        },
+        "baseUserDN": {
+          "description": "The base distinguished name for users",
+          "type": "string"
+        },
+        "baseGroupDN": {
+          "description": "The base distinguished name for groups",
+          "type": "string"
+        },
+        "primaryServer": {
+          "description": "Primary server URL",
+          "type": "string"
+        },
+        "secondaryServer": {
+          "description": "Secondary server URL",
+          "type": "string"
+        },
+        "ssl": {
+          "description": "Protect LDAP communication using SSL certificate (LDAPS)",
+          "type": "string",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "x-ms-enum": {
+            "name": "SslEnum",
+            "modelAsString": true
+          }
+        },
+        "username": {
+          "description": "The ID of an Active Directory user with a minimum of read-only access to Base DN for users and group",
+          "type": "string"
+        },
+        "password": {
+          "description": "The password of the Active Directory user with a minimum of read-only access to Base DN for users and groups.",
+          "type": "string"
+        }
+      }
+    },
+    "PrivateCloud": {
+      "description": "A private cloud resource",
+      "allOf": [
+        {
+          "$ref": "#/definitions/TrackedResource"
+        }
+      ],
+      "properties": {
+        "sku": {
+          "description": "The private cloud SKU",
+          "$ref": "#/definitions/Sku"
+        },
+        "properties": {
+          "description": "The properties of a private cloud resource",
+          "$ref": "#/definitions/PrivateCloudProperties"
+        }
+      }
+    },
+    "PrivateCloudProperties": {
+      "description": "The properties of a private cloud resource",
+      "properties": {
+        "provisioningState": {
+          "description": "The provisioning state",
+          "type": "string",
+          "readOnly": true,
+          "enum": [
+            "Succeeded",
+            "Failed",
+            "Cancelled",
```

Comment:
>Cancelled [](start = 13, length = 9)

Please change to `Canceled` which is the official spelling of that state value. ([https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/Addendum.md#provisioningstate-property](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/Addendum.md#provisioningstate-property)) There may be state machines in SDKs and clients that take a dependency on the exact spelling of that state.

Reply:
This API version is implemented and shipped. I can't change it for this version, but we can look into it for 2020-03-20-preview.

Reply:
Out of curiosity, does the generated SDK actually work with the API as-is? If it works fine, then there's no issue.

---
In reply to: [419677989](https://github.com/Azure/azure-rest-api-specs/pull/9307#discussion_r419677989) [](ancestors = 419677989)

Reply:
We use the value in swagger so SDKs should work fine.